modules/rabbitmq/rabbitmq_test.go

```diff
@@ -216,27 +212,13 @@ func TestRunContainer_withAllSettings(t *testing.T) {
 	testcontainers.CleanupContainer(t, rabbitmqContainer)
 	require.NoError(t, err)
 
-	if !assertEntity(t, rabbitmqContainer, "queues", "queue1", "queue2", "queue3", "queue4") {
-		t.Fatal(err)
-	}
-	if !assertEntity(t, rabbitmqContainer, "exchanges", "direct-exchange", "topic-exchange", "topic-exchange-2", "topic-exchange-3", "topic-exchange-4") {
-		t.Fatal(err)
-	}
-	if !assertEntity(t, rabbitmqContainer, "users", "user1", "user2") {
-		t.Fatal(err)
-	}
-	if !assertEntity(t, rabbitmqContainer, "policies", "max length policy", "alternate exchange policy") {
-		t.Fatal(err)
-	}
-	if !assertEntityWithVHost(t, rabbitmqContainer, "policies", 2, "max length policy", "alternate exchange policy") {
-		t.Fatal(err)
-	}
-	if !assertEntity(t, rabbitmqContainer, "operator_policies", "operator policy 1") {
-		t.Fatal(err)
-	}
-	if !assertPluginIsEnabled(t, rabbitmqContainer, "rabbitmq_shovel", "rabbitmq_random_exchange") {
-		t.Fatal(err)
-	}
+	require.True(t, assertEntity(t, rabbitmqContainer, "queues", "queue1", "queue2", "queue3", "queue4"))
+	require.True(t, assertEntity(t, rabbitmqContainer, "exchanges", "direct-exchange", "topic-exchange", "topic-exchange-2", "topic-exchange-3", "topic-exchange-4"))
+	require.True(t, assertEntity(t, rabbitmqContainer, "users", "user1", "user2"))
+	require.True(t, assertEntity(t, rabbitmqContainer, "policies", "max length policy", "alternate exchange policy"))
+	require.True(t, assertEntityWithVHost(t, rabbitmqContainer, "policies", 2, "max length policy", "alternate exchange policy"))
+	require.True(t, assertEntity(t, rabbitmqContainer, "operator_policies", "operator policy 1"))
+	require.True(t, assertPluginIsEnabled(t, rabbitmqContainer, "rabbitmq_shovel", "rabbitmq_random_exchange"))
 }
 
 func assertEntity(t *testing.T, container testcontainers.Container, listCommand string, entities ...string) bool {
```

Comment:
We can probably rename this function to `requireEntity` and inside call require 🤔 It would simplify the above block with just plain calls to the helper function

Reply:
Whatever you prefer is fine for me: I see connections in the change, but I do see it as a separate change. In any case fine.